src/ciphersuite.rs

```diff
@@ -19,8 +20,10 @@ pub trait CipherSuite {
     /// an extension trait PasswordToCurve that allows some customization on
     /// how to hash a password to a curve point. See `group::Group`.
     type Group: Group;
+    /// A `Group` used for the `KeyExchange`.
+    type AkeGroup: Group;
```

Comment:
Perhaps can we rename this type to KeGroup instead of AkeGroup?

And then we could also consider renaming the type on line 22 from "Group" to "OprfGroup"... though I am less confident about that one.

Reply:
I think it's easier for the consumer if we rename to `OprfGroup`, even if the naming might not align to wording from the spec.